scripts/add-task.js

```diff
@@ -11,14 +12,14 @@ function addTask(
   taskListElement
 ) {
   createButton.addEventListener(CLICK_EVENT, () => {
```

Comment:
event listener registered with an arrow/anonymous function won't be cleaned up, consider introducing a named function so it can be unregistered.